docs/source/ucx.rst

```diff
@@ -1,135 +1,199 @@
-- ``--rmm-pool-size`` equivalent to ``DASK_RMM__POOL_SIZE``.
+UCX configuration options will need to be specified for ``dask-scheduler`` as environment variables; see `Dask Configuration <https://docs.dask.org/en/latest/configuration.html#environment-variables>`_ for more details on the mapping between environment variables and options.
 
-Here's how to start workers with all transports that are currently relevant for us:
+To start a Dask scheduler using UCX with all supported transports and a 1 gigabyte RMM pool:
 
 .. code-block:: bash
 
-    dask-cuda-worker ucx://SCHEDULER_IB0_IP:8786 --enable-tcp-over-ucx --enable-nvlink --enable-infiniband -- enable-rdmacm --net-devices="auto" --rmm-pool-size="30GB"
+    DASK_UCX__CUDA_COPY=True \
+    DASK_UCX__TCP=True \
+    DASK_UCX__NVLINK=True \
+    DASK_UCX__INFINIBAND=True \
+    DASK_UCX__RDMACM=True \
+    DASK_UCX__NET_DEVICES=mlx5_0:1 \
+    DASK_RMM__POOL_SIZE=1GB \
+    dask-scheduler --protocol ucx --interface ib0
 
+Note the specification of ``mlx5_0:1`` as our UCX net device; because the scheduler does not rely upon Dask-CUDA, it cannot automatically detect InfiniBand interfaces, so we must specify one explicitly *(is this true?)*.
+We communicate to the scheduler that we will be using UCX with the ``--protocol`` option, and that we will be using InfiniBand with the ``--interface`` option.
 
-client
-^^^^^^
+To start the same Dask scheduler as above but only using NVLink:
 
-The same configurations used for the scheduler should be used by the client. One possible exception is ``DASK_RMM__POOL_SIZE``, at this time it's unclear whether this is necessary or not, but using that should not cause any issues nevertheless.
+.. code-block:: bash
 
-One can use ``os.environ`` inside the client script, it's important to set them at the very top before importing anything other than ``os``. See example below:
+    DASK_UCX__CUDA_COPY=True \
+    DASK_UCX__TCP=True \
+    DASK_UCX__NVLINK=True \
+    DASK_RMM__POOL_SIZE=1GB \
+    dask-scheduler --protocol ucx --interface eth0
 
-.. code-block:: python
+Note that we no longer specify a net device, as this generally can be skipped when using a non-InfiniBand interface.
 
-    import os
+Workers
+"""""""
 
-    os.environ["DASK_RMM__POOL_SIZE"] = "1GB"
-    os.environ["DASK_UCX__CUDA_COPY"] = "True"  # os.environ needs using strings, not Python True/False
-    os.environ["DASK_UCX__TCP"] = "True"
-    os.environ["DASK_UCX__NVLINK"] = "True"
-    os.environ["DASK_UCX__INFINIBAND"] = "True"
-    os.environ["DASK_UCX__NET_DEVICES"] = "mlx5_0:1"
+All the relevant Dask configuration options for UCX have analogous parameters in ``dask-cuda-worker``; see :doc:`Worker <worker>` for a complete list of these options.
 
-    from distributed import Client
+To start workers with all supported transports and a 1 gigabyte RMM pool *(do we need to specify an interface here?)*:
```

Comment:
In most cases, no, UCX will be capable of identifying the optimal network interface to use.